intlogger.go

```diff
@@ -353,20 +377,36 @@ func (l *intLogger) logPlain(t time.Time, name string, level Level, msg string,
 				}
 			}
 
-			var key string
-
+			// Convert the field key to a string.
 			switch st := args[i].(type) {
 			case string:
 				key = st
 			default:
 				key = fmt.Sprintf("%s", st)
 			}
 
+			// Optionally apply the ANSI "faint" and "bold"
+			// SGR values to the key.
+			if l.fieldColor != ColorOff {
+				key = faintBoldColor.Sprint(key)
+			}
+
+			// Values may contain multiple lines, and that format
+			// is preserved, with each line prefixed with a "  | "
+			// to show it's part of a collection of lines.
+			//
+			// Values may also need quoting, if not all the runes
+			// in the value string are "normal", like if they
+			// contain ANSI escape sequences.
 			if strings.Contains(val, "\n") {
 				l.writer.WriteString("\n  ")
 				l.writer.WriteString(key)
 				l.writer.WriteString("=\n")
-				writeIndent(l.writer, val, "  | ")
+				if l.fieldColor != ColorOff {
+					writeIndent(l.writer, val, faintColor.Sprint("  | "))
```

Comment:
One super minor thing, since this is constant, can we pull the `faintColor.Sprint(" |")` out to a global var?

Reply:
No problem can do 👍

Reply:
Should be addressed with https://github.com/hashicorp/go-hclog/commit/fd264152857468feff6a784c047e6585bebdd664